installer/parse-install-yaml.py

```diff
@@ -31,19 +32,32 @@ def main():
 
         try:
             install_type = install_item["type"]
-            if install_type == "ros":
-                source_type = install_item["source"]["type"]
+            if install_type == "empty":
+                return 0
+            elif install_type == "ros":
+                if "source" in install_item:
+                    source = install_item["source"]
+                else:
+                    release = environ["ROS_DISTRO"]
```

Comment:
```suggestion
                    release = environ["TUE_ROS_DISTRO"]
```
More consistent with rest of the installer

Reply:
Suits me, but in which case are they different?

Reply:
I don’t think they will be different at any time. But TUE_ROS_DISTRO, will probably be set before ROS_DISTRO, so definitely always available.